public/app/percona/shared/helpers/validators.ts

```diff
@@ -12,7 +12,11 @@ export const validators = {
 
     return 'Port should be a number and between 0 and 65535';
   },
+  validateUrl: (value: string) => {
+    const urlRe = /(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/;
```

Comment:
Not sure we'll ever need ftp, or we'll add it when needed.

Reply:
Alex, thanks, copypaste power :D removed FTP from regexp